theme/src/components/header.js

```diff
@@ -138,49 +137,21 @@ Header.defaultProps = {
   isSearchEnabled: true
 }
 
-function PrimerNavItems({siteMetadata, items}) {
+function PrimerNavItems({siteMetadata, items, path}) {
   return (
     <>
       <VisuallyHidden>
         <h3 aria-labelledby="site-header">{siteMetadata.header.title} </h3>
       </VisuallyHidden>
-      <Box
-        as={'nav'}
-        sx={{display: 'flex', alignItems: 'center', justifyContent: 'space-between', color: 'fg.default', gap: 2}}
-      >
+      <UnderlineNav aria-label="top navigation" sx={{border: 'none'}}>
         {items.map((item, index) => {
-          if (item.children) {
-            return (
-              <Box key={index}>
-                <NavDropdown title={item.title}>
-                  {item.children.map(child => (
-                    <NavDropdownItem key={child.title} href={child.url}>
-                      {child.title}
-                    </NavDropdownItem>
-                  ))}
-                </NavDropdown>
-              </Box>
-            )
-          }
-
           return (
-            <Link
-              key={index}
-              href={item.url}
-              sx={{
-                display: 'block',
-                color: 'fg.default',
-                fontSize: 2,
-                fontWeight: 'bold',
-                ml: 2,
-                mr: 2
-              }}
-            >
+            <UnderlineNav.Link key={index} href={item.url} selected={item.url === path}>
```

Comment:
`item.url === path` may return false if `item.url` is something like `"https://primer.style/brand"`, and `path` is something like `"/brand"`.

I'm not sure if this is _actually_ an issue, but it's something to be aware of when we deploy to prod.

Reply:
You're right. Should we go with something like this instead?

```js
selected = {item.url === siteMetadata.header.url + path }
```

Reply:
Let's try it! Maybe we can test this by opening a draft PR in primer/design so we can test a preview deployment?

Reply:
preview deployments generate unique domains like `https://primer-6f994bf089-26493675.drafts.github.io/` so I don't think using the `header.url` will work 😱 . I think there's only one way to know 🚢 🤣

Reply:
Ok, let's do it :shipit: 🤞